### PR TITLE
1655/invite user edit

### PR DESCRIPTION
**Pull request by dominikx96**
# Pull Request Template

## Issue

Addresses #1655

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Adds the ability to edit a user, delete and resend an invite.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

1. Create a new user, activate it using a link from the email, and try to log in
2. Create a user and do not activate it. Instead of - click on the "resend confirmation" button and activate using a new link
2. Update user data and check the user summary again
3. Delete a user and check the users table 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [x] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

**Comment by seanmalbert** [comment]
@dominikx96 ,
It means that the jurisdiction passed in the request isn't defined. In `backend/core/src/jurisdictions/services/jurisdiction-resolver.service.ts`, it gets `jurisdictionName` from the request and then searches jurisdictions by the name. If there is no result, then the jurisdiction isn't in the database.

File: sites/partners/pages/users/index.tsx [hunk 1]
```diff
@@ -11,17 +11,25 @@ import {
...
 }
 
+type IsDrawerOpenValue = {
+  type: "add" | "edit"
+  user?: User
+}
+
 const Users = () => {
+  /* Add user drawer */
+  const [isDrawerOpen, setDrawerOpen] = useState<IsDrawerOpenValue | null>(null)
```
[the remark this answers is not in the document]
Comment (jaredcwhite) [reply sites/partners/pages/users/index.tsx:31]: @dominikx96 Maybe just `userDrawer`/`setUserDrawer`?

**Review by seanmalbert — APPROVED** [review]
@dominikx96 ,
I updated the jurisdiction resolver service to return undefined if no jurisdictionName is present in the request, so resending the confirmation now works.

There are some missing translation strings that should be added, like `users.editUser`, `users.userUpdated` and `users.resendinvite`, can you please resolve those.

**Comment by dominikx96** [comment]
@seanmalbert these translations are defined, I've just added translation for delete modal. I think everything should be in a good shape, and we can merge it after final @jaredcwhite review :)

**Pull request merged**